README.md

```diff
@@ -9,9 +9,6 @@ Android App to export GPX and FIT to garmin devices
 * see Wiki: https://github.com/gimportexportdevs/gexporter/wiki/Help
 
 ## HOWTO Develop
-* create directory app/libs
-* copy fit.jar from the [Garmin FitSDK](https://developer.garmin.com/fit/download/) to app/libs
-* Download the [Connect IQ Mobile SDK (Android BLE/ADB)](https://developer.garmin.com/connect-iq/sdk/) and copy the .aar file to app/libs
```

Comment:
I added the dependencies so one does not have to download oneself.

I don't know if you need the files locally for some other kind of debugging. If so I can leave the documentation unchanged.